fern/ir-types-v29/definition/types.yml

```diff
@@ -266,7 +271,7 @@ types:
         type: double
         key: double
       string:
-        type: string
+        type: commons.SafeAndUnsafeString
```

Comment:
`EscapedExampleString`

Reply:
Decision: turn into object for now

```yaml
   wireString: 
```

Reply:
I went with the following structure, using `original` instead of `wireString` because these values aren't necessarily sent on the wire.

```yaml
  EscapedString:
    docs: |
      Defines the original string, and its escaped-equivalent (depending on the target programming language).
      This is paricularly relevant to example string literals.
      For example, in Python we escape strings that contain single or double quotes by using triple quotes,
      in Go we use backticks, etc.
    properties:
      original: string
```

Let me know what you think!

Reply:
sounds great